### PR TITLE
Rs 2022.09.25.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-09-25-222403_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 1]
```diff
@@ -26,50 +27,49 @@ And on the other hand――
...
-Maybe this is a bit unusually soft by me too,
- but I just don't want to see her make that kind of face, no matter the reason.
+Maybe I've started to go a bit soft too.
+ Regardless of the reason, I just can't stand to see her make that kind of face.
 }
 [sha:9a8d150a66ae9a3728a80d18381adb2a1c9da5bf]{
 -- Page 626, Offset 16311.
 -- 「いい、結果的には誰にも見つからなかったし。
-"It's fine, thanks to that nobody found us out...
+"It's fine, at the end of the day we didn't get caught.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:46]: day, we didn't

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 2]
```diff
@@ -79,12 +79,12 @@ But it's true you took things too far.
...
 -- Page 628, Offset 16317. Glued. Mods: @x, @k, @e.
 -- ええ、任せて！
- Sure, I gotcha!
+ Sure, you got it!
 }
 [sha:93005a9a44f70ac2415e57b8409e924a99aeb557]{
 -- Page 628, Offset 16318. Glued. Mods: @x.
 -- 協力無比のコンビだものね！」
- We're a combo whose teamwork is without equal, after all!"
+ We make an incomparably strong team together, after all!"
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:87]: "incomparably" is a liiiitle stiff but I think it's fine.
drop "together"

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 3]
```diff
@@ -106,12 +106,14 @@ For a second, I was worried she meant to say 'whose power is without equal' ther
 [sha:997309b2db3591847212f1ed71b94bdff9187a64]{
 -- Page 629, Offset 16322.
 -- 一息ついた事で油断したのか、一瞬、意識が消えかかった。
-Maybe the breather made me a bit careless, because for a split second, I get close to losing consciousness.
+I must have let my guard down after we stopped for a breather,
+ because for a split second I feel like I'm about to black out.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:110]: second, I feel

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 4]
```diff
@@ -270,42 +280,42 @@ Of course not. I don't want to see something like that ever again.
...
 -- Page 643, Offset 16357.
 -- おまえはそのままが一番だと思うよ」
-I think it'd be best if you could always be like that."
+I think you're at your best when you're like that."
 }
 [sha:b419f6400b14f78a096d1813ad10b1d32bd07af3]{
 -- Page 643, Offset 16358.
 -- 叶うのなら、この笑顔をいつまでも見ていたい。
-And if possible, I wish I could look upon that smile forever.
+And if I could make one wish, it would be that I could look upon that smile forever.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:308]: ciel route btw

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 5]
```diff
@@ -351,44 +362,45 @@ And yet―――
...
 -- 「はっ、ァ―――……ハ、ァ……」
 // tried to match it with the VO
-"Ugh, hahhh... hahhh..."
+"%{i}Ugh%{/i}, %{i}hahhh%{/i}... %{i}hahhh%{/i}..."
 }
 [sha:14cc0b8c21f7c9b5d68f28b48b9e70d93665a4b1]{
 -- Page 647, Offset 16373.
 -- 大きく息を吸って目眩をやり過ごす。
-I breathe deeply as to get over my dizziness.
+I take a few deep breaths and let the dizziness pass over me.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:387]: I'd just leave it at "dizziness pass."

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 6]
```diff
@@ -152,39 +156,44 @@ Shiki, how much do you understand about your eyes?"
...
 -- Page 633, Offset 16332.
 -- ……アルクェイドによると、俺の眼は死を視ているのではなく、死を解読しているようなものなんだとか。
-According to Arcueid, my eyes aren't %{i}seeing%{/i} death, as much as they're %{i}deciphering%{/i} it,
+According to Arcueid, my eyes aren't %{i}seeing%{/i} death so much as %{i}deciphering%{/i} it.
 }
 [sha:50c6641cbb94a49bab14aa6283f5a85a692314c9]{
 -- Page 633, Offset 16333.
 -- 『線』はあくまで死の露出であり、その線の基点である『点』こそが死そのもの。
-Ultimately, the lines are just death laid bare, while the origination of those lines――the points――are death itself.
+// Most uses of 露出 that I turned up seem to be exposure in the context of photography (rather than to bare something),
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:169]: Meanwhile the context I see this word in most is porn... sure explains my interpretation...

File: script/Ciel/Day 7/07_01_CIEL07_8.txt [hunk 7]
```diff
@@ -437,50 +449,50 @@ The time passes as we talk about this and that, and before I know it, it has tur
...
 -- Page 656, Offset 16389.
 -- 涼しげに笑って、アルクェイドは公園から跳び去っていった。
-She flashes a nonchalant smile, before leaping away from the park.
+Arcueid flashes me a nonchalant smile and leaps away from the park.
 }
 [sha:fe2edcf078a84bcc0e8dc555e6bebc86ab44c185]{
 -- Page 656, Offset 16390.
 -- 登場も退場も待ったなしの女である。
-This woman leaves as suddenly as she appears.
+She sure doesn't waste any time when making an entrance. Or exit, for that matter.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_8.txt:480]: I don't really like this sentence, but I don't like the original much either.

**Review by Petrikow — COMMENTED** [review]
otu

**Pull request merged**